decidim-meetings/lib/decidim/api/meeting_type.rb

```diff
@@ -9,33 +9,43 @@ class MeetingType < Decidim::Api::Types::BaseObject
       field :agenda, Decidim::Meetings::AgendaType, "Agenda for this meeting, if available", null: true
       field :attendee_count, GraphQL::Types::Int, "Amount of attendees to this meeting", method: :attendees_count, null: true
       field :attending_organizations, GraphQL::Types::String, "list of attending organizations", null: true
       field :audio_url, GraphQL::Types::String, "URL for the audio of the session, if any", null: true
       field :closed, GraphQL::Types::Boolean, "Whether this meeting is closed or not.", method: :closed?, null: false
+      field :closed_at, Decidim::Core::DateTimeType, "The date and time this meeting was closed", null: true
       field :closing_report, Decidim::Core::TranslatedFieldType, "The closing report of this meeting.", null: true
+      field :comments_enabled, GraphQL::Types::Boolean, "Whether the comments are enabled or not", null: false
+      field :comments_end_time, Decidim::Core::DateTimeType, "The ending time when the comments will be allowed on this meeting", null: true
+      field :comments_start_time, Decidim::Core::DateTimeType, "The starting time when the comments will be allowed on this meeting", null: true
       field :contribution_count, GraphQL::Types::Int, "Amount of contributions to this meeting", method: :contributions_count, null: true
-      field :coordinates, Decidim::Core::CoordinatesType, "Physical coordinates for this meeting", null: true
+      field :customize_registration_email, GraphQL::Types::Boolean, "The email to customize for the registration", null: true
       field :description, Decidim::Core::TranslatedFieldType, "The description of this meeting.", null: true
       field :end_time, Decidim::Core::DateTimeType, "The time this meeting ends", null: false
       field :id, GraphQL::Types::ID, "ID of this meeting", null: false
+      field :iframe_access_level, GraphQL::Types::String, "The iframe access level of the meeting ", null: true
       field :iframe_embed_type, GraphQL::Types::String, "The type of displaying of the online meeting URL", null: true
       field :is_withdrawn, GraphQL::Types::Boolean, "Whether this meeting is withdrawn or not.", method: :withdrawn?, null: false
       field :location, Decidim::Core::TranslatedFieldType, "The location of this meeting (free format)", null: true
       field :location_hints, Decidim::Core::TranslatedFieldType, "The location of this meeting (free format)", null: true
       field :online_meeting_url, GraphQL::Types::String, "The URL of the meeting (when the type is online)", null: false
       field :private_meeting, GraphQL::Types::Boolean, "Whether the meeting is private or not (it can only be true if transparent)", null: false
-      field :reference, GraphQL::Types::String, "Reference for this meeting", null: false
+      field :public_participants, [Decidim::Core::UserType, { null: true }], "The object's services", null: false
```

Comment:
This description is wrong

Reply:
changed to "The list of users participating to this meeting"